pptx/dml/color.py

```diff
@@ -91,6 +91,25 @@ def theme_color(self, mso_theme_color_idx):
             self._color = _SchemeColor(schemeClr)
         self._color.theme_color = mso_theme_color_idx
 
+    @property
+    def preset_color(self):
+        """  Preset Color value of this color.
+        
+        Pone of those defined in the MSO_PRESET_COLOR_INDEX enumeration.  
```

Comment:
Type on first word, I believe?

Reply:
Yup, and actually that is incorrect comments now as I ended up not doing a proper ENUM like it says here.  Updated